app/src/sharedTest/java/org/simple/clinic/TestData.kt

```diff
@@ -1226,7 +1227,7 @@ object TestData {
       gender: String = "M",
       state: String = "Maharashtra",
       district: String = "Thane",
-      dateOfBirth: String = "12/12/1997",
+      dateOfBirth: LocalDate = LocalDate.parse("12/12/1997", DateTimeFormatter.ofPattern("d/M/yyyy")),
```

Comment:
What's the need to create a formatter here? Using `LocalDate.parse` should be sufficient, right?

Reply:
No. I was getting `DateTimeParseException` when I was going that as the default local date pattern is `yyyy-MM-dd`

Reply:
Yeah, this means that you are passing the date in the wrong format to `LocalDate.parse`. This version of the factory method should only be used when the date string is coming from a source that you don't have control over, which is not the case here.